src/phoenix/datasets/types.py

```diff
@@ -183,3 +208,196 @@ def __post_init__(self) -> None:
     Callable[[Example], TaskOutput],
     Callable[[Example], Awaitable[TaskOutput]],
 ]
+
+
+@dataclass(frozen=True)
+class ExperimentConfig:
+    n_examples: int
+    n_repetitions: int = 1
+
+    @property
+    def count(self) -> int:
+        return self.n_examples * self.n_repetitions
+
+
+@dataclass(frozen=True)
+class EvaluationConfig:
+    eval_names: FrozenSet[str]
+    exp_config: ExperimentConfig
+
+
+@dataclass(frozen=True)
+class _HasStats:
+    _title: str = field(repr=False, default="")
+    stats: pd.DataFrame = field(repr=False, default_factory=pd.DataFrame)
+    timestamp: datetime = field(repr=False, default_factory=lambda: datetime.now(timezone.utc))
+
+    @property
+    def title(self) -> str:
+        return f"{self._title} ({self.timestamp:%x %X})"
+
+    def __str__(self) -> str:
+        try:
+            assert int(version("pandas").split(".")[0]) >= 1
+            # `tabulate` is used by pandas >= 1.0 in DataFrame.to_markdown()
+            import tabulate  # noqa: F401
+        except (AssertionError, ImportError):
+            text = self.stats.__str__()
+        else:
+            text = self.stats.to_markdown(index=False)
+        return f"{self.title}\n{'-'*len(self.title)}\n" + text
+
+
+@dataclass(frozen=True)
+class EvaluationSummary(_HasStats):
+    _title: str = "Evaluation Summary"
```

Comment:
```suggestion
    _title: str = "Experiment Summary"
```

Reply:
I guess it really doesn't matter.

Reply:
i actually split this into a separate display sections because the columns headers are different